examples/sensor-data-scala/README.md

```diff
@@ -49,23 +49,24 @@ wrk -c 400 -t 400 -d 500 -s wrk-04-moderate-breeze.lua <http-ingress-url>
 ```
 
 
-### Example Deployment on GKE
+### Example Deployment on Kubernetes
 
-* Make sure you have installed a GKE cluster with Cloudflow running as per the [installation guide](https://github.com/lightbend/cloudflow-installer).
-Make sure you have access to your cluster:
+* Make sure you have installed a Kubernetes cluster with Cloudflow running as per the [installation guide](https://github.com/lightbend/cloudflow-installer).
+
+* Verify GKE cluster and Google docker registry 
 
 ```
 $ gcloud container clusters get-credentials <CLUSTER_NAME>
+$ gcloud auth configure-docker
 ```
 
-and that you have access to the Google docker registry:
-
+* Verify EKS cluster and ECR docker registry
 ```
-$ gcloud auth configure-docker
+$ aws eks describe-cluster <CLUSTER_NAME>
+$ eval $(aws ecr get-login --no-include-email --region us-east-1)
```

Comment:
More instructions are needed here. In order to publish the app image I had to use the registry name 
printed from `aws ecr get-login --no-include-email --region us-east-1`, which is the default one for the aws account being used. Next I run:
```aws ecr create-repository --repository-name  stavros-test/sensor-data-scala```
and finally set:
``` ThisBuild / cloudflowDockerRegistry := Some("account_number.dkr.ecr.eu-west-1.amazonaws.com")
ThisBuild / cloudflowDockerRepository := Some("stavros-test")```

Reply:
You are right on that, I was debating if we should be using ECR or just Docker Hub. Happy to add those instructions...

Reply:
Let's use the Cloud specific registries for now.

Reply:
Yes pls do so.